tools/buildsys/src/manifest.rs

```diff
@@ -233,6 +233,15 @@ when the platform supports it.
 uefi-secure-boot = true
 ```
 
+`fips` means that FIPS-certified modules will be used for cryptographic operations. This affects
+the kernel at runtime. It also causes alternate versions of Go and Rust programs that use
```

Comment:
I am kind of over-promising here since I'm still not 100% sure what the Rust integration will look like.

Reply:
Just for arguement's sake and clarification, is this below Cargo.toml add for the variant/rpm package or is it for rust code, and if for rust code is there a reason not to translate it to a standardized feature flag?

Reply:
> Just for arguement's sake and clarification, is this below Cargo.toml add for the variant/rpm package or is it for rust code, and if for rust code is there a reason not to translate it to a standardized feature flag?

It's for the buildsys-related metadata for variants. I'd like to keep it separate from Cargo's own functionality because - as distinctive and delightful as Bottlerocket's use of `cargo` is - eventually I expect to create our own tool to do what it does for us now (tracking changes, traversing the dependency graph, driving builds).